### ISSUE TITLE
Vehicle List Multi Selection

### ISSUE THREAD

**Issue by TrevorCash** [OpenLoco/OpenLoco#1002]
The ability to select multiple vehicles on the vehicle list might be nice (maybe a checkbox to the left of each train? as well as highlighting multiple). And then on the bottom of the window there could be region where there are buttons that do actions on the vehicles you have selected. for example deleting, de-railing, stopping, setting route, etc. Any actions that could be applied to multiple trains.

**Comment by luk3ZPL** [comment]
What about grouping vehicles and add actions for all in group ? Similar to: https://wiki.openttd.org/en/Manual/Vehicle%20groups

**Comment by TrevorCash** [comment]
@luk3ZPL Yeah managing trains by groups (and potentially group hierarchies) would definitly be on my wish list.

**Comment by LeftofZen** [comment]
> What about grouping vehicles and add actions for all in group ? Similar to: https://wiki.openttd.org/en/Manual/Vehicle%20groups

See https://github.com/OpenLoco/OpenLoco/issues/1440
